saliency/blur_ig_test.py

```diff
@@ -16,22 +16,82 @@
 """
 from . import blur_ig
 import numpy as np
+from tensorflow import test
 import tensorflow.compat.v1 as tf
-from tensorflow.compat.v1.python.platform import googletest
 
+OUTPUT_GRADIENTS = blur_ig.OUTPUT_GRADIENTS
 
-class BlurIgTest(googletest.TestCase):
-  """
-  To run:
-  "python -m saliency.blur_ig_test" from the PAIR-code/saliency directory.
-  """
 
+class BlurIgTest(test.TestCase):
+  """To run: "python -m saliency.blur_ig_test" top-level saliency directory."""
+
+  def create_call_model_function(self, session, grad_node, x):
+    def call_model(x_value_batch, call_model_args={}, expected_keys=None):
+      call_model.num_calls += 1
+      call_model_args[x] = x_value_batch
+      data = session.run(grad_node, feed_dict=call_model_args)
+      return {OUTPUT_GRADIENTS: data[0]}
```

Comment:
Why is only the first index returned?

Reply:
This is a good catch. We should probably add assert statements when the output is returned, to make sure it is the same shape as the input.

Reply:
Added additional tests